### PR TITLE
fix: CVE vulerabilite mitigation

### PR DESCRIPTION
**Pull request by snoopy82481**

### CODE REVIEW

File: src/Directory.Build.targets [hunk 1]
```diff
@@ -44,6 +44,13 @@
 
   </ItemGroup>
 
+  <!-- Following found during vulerabilities Code Scan  -->
+  <ItemGroup>
+    <PackageReference Update="System.Net.Http" Version="4.3.4" />
+    <PackageReference Update="System.Private.Uri" Version="4.3.2" />
+    <PackageReference Update="System.Net.RegularExpressions" Version="4.3.1" />
```
Comment (rcdailey) [inline src/Directory.Build.targets:51]: This should be `System.Text.RegularExpressions`, based on your report in #112

File: src/Recyclarr/Recyclarr.csproj [hunk 2]
```diff
@@ -16,6 +16,13 @@
     <PackageReference Include="YamlDotNet" />
   </ItemGroup>
 
+  <!-- Following found during vulerabilities Code Scan  -->
+  <ItemGroup>
+    <PackageReference Include="System.Net.Http" />
+    <PackageReference Include="System.Private.Uri" />
+    <PackageReference Include="System.Net.RegularExpressions" />
```
Comment (rcdailey) [inline src/Recyclarr/Recyclarr.csproj:23]: As mentioned in the other comment, this will need to change to `System.Text.RegularExpressions` as well.

**Review by rcdailey — CHANGES_REQUESTED** [review]

File: src/Directory.Build.targets (hunk 1 above)
Comment (snoopy82481) [reply src/Directory.Build.targets]: I fixed this in the latest push

File: src/Recyclarr/Recyclarr.csproj (hunk 2 above)
Comment (snoopy82481) [reply src/Recyclarr/Recyclarr.csproj]: I fixed this in the latest push

**Review by rcdailey — APPROVED** [review]

**Comment by rcdailey** [comment]
Thanks for the changes. Don't worry about the Sonarcloud failure; it shouldn't be running on pull requests because it requires github secrets that won't be available.

**Pull request merged**